### PR TITLE
Adding instruction + template for facilitator prep before story session

### PR DESCRIPTION
**Pull request by amcasari**
1. Added in a checklists w/ notes on prepping for facilitating a session
2. Added a prep "session flow" doc w/ spaces for personal information about the storyteller + bio notes

### CODE REVIEW

File: facilitator-prep.md
Comment (juliaferraioli) [inline facilitator-prep.md:19]: We should probably update this doc to reflect that sessions are now half the time.

File: facilitator-prep.md
Comment (juliaferraioli) [inline facilitator-prep.md:29]:
```suggestion
* Have you checked your audio recording channels? We recommend (if possible) capturing each audio stream separately.
```

File: session-prep-template.md
Comment (juliaferraioli) [inline session-prep-template.md:3]: This is all captured in the Calendly invitation, so we don't need the facilitator to re-ask the storyteller to provide this information.

File: session-prep-template.md
Comment (juliaferraioli) [inline session-prep-template.md:27]: 😆

File: session-prep-template.md
Comment (juliaferraioli) [inline session-prep-template.md:35]: Extraneous blank line

**Review by juliaferraioli — CHANGES_REQUESTED** [review]
Some suggestions.

File: facilitator-prep.md
Comment (amcasari) [reply facilitator-prep.md:19]: Updated w/ 
* new question ratio suggestions (15-mins + 30-mins)
* split out the intro + warm-up question
* started a section for warm-up question
* created [a new issue in story-questions]((https://github.com/opensourcestories/story-questions/issues/3)) to remind myself we need that bank

File: session-prep-template.md
Comment (amcasari) [reply session-prep-template.md:3]: Yes. I was moving our Google Doc template that we use to prep before a session into a template here. 

Should the change here be an additional annotation in the template as well as a change in wording to the facilitator checklist?

Comment (juliaferraioli) [reply session-prep-template.md:3]: I think the updates you made resolve my concern.

File: session-prep-template.md
Comment (amcasari) [reply session-prep-template.md]: 🤦‍♀️

**Comment by amcasari** [comment]
Thanks @juliaferraioli! I'm going to give @Ruth-ikegah a chance to check this PR against our recent conversation. Ruth, no rush.

**Review by juliaferraioli — APPROVED** [review]
LGTM but waiting for @Ruth-ikegah to review before merging.

**Review by Ruth-ikegah — APPROVED** [review]
LGTM too! Sorry for the delay!

**Pull request merged**
